src/core/layout/layout.js

```diff
@@ -150,7 +128,7 @@ export function create(scene, metrics, buttonIds) {
         addToGroup,
         buttons,
         debug,
-        getSafeArea,
+        getSafeArea: getSafeArea(groups),
```

Comment:
Just to clarify, this will return a func for the original GelGroups? Any groups added after, are not used I guess, guess thats fine

Reply:
Yeah, but I think custom groups would be unlikely to be part oif the calculation.

On thought -  do you think it would be clearer if I called the function in safe-area-js "getSafeAreaFn" ?

so that line would look like: getSafeArea: getSafeAreaFn(groups),

Or does that make it worse?

Reply:
I guess it may be a bit clearer that its returning a function, yea

Reply:
Cool - have updated, at least illustrates that the `getSafeArea` hanging off layout is a different thing to the one imported from safe-area.js